examples/mnist-classification/project.clj

```diff
@@ -0,0 +1,20 @@
+(defproject mnist-classification "0.9.4-SNAPSHOT"
+  :description "An example of using experiment/classification on mnist."
+  :dependencies [[org.clojure/clojure "1.8.0"]
+                 [thinktopic/experiment "0.9.4-SNAPSHOT"]
```

Comment:
Lets not depend on a snapshot version in examples so people can run the examples with the code thats coming from clojars.